templates/patterns/navbar/pattern-navbar.html.twig

```diff
@@ -23,15 +23,11 @@
 {% endblock %}
 
 {% block navigation %}
-  <div class="me-auto navbar-nav">
     {{ navigation }}
-  </div>
 {% endblock %}
 
 {% block right %}
   {% if right is not empty %}
-    <div class="d-flex">
-      {{ right }}
-    </div>
+    {{ right }}
   {% endif %}
```

Comment:
We no longer need `if right is not empty`, now that we have no more wrappers.
Also check the similar cases above.